cmd/root.go

```diff
@@ -110,6 +110,8 @@ func init() {
 	RootCmd.PersistentFlags().String("pass", "", "Password to use to authenticate (If empty will look for EC_PASS environment variable)")
 	RootCmd.PersistentFlags().String("api-key", "", "API key to use to authenticate (If empty will look for EC_API_KEY environment variable)")
 	RootCmd.PersistentFlags().Bool("verbose", false, "Enable verbose mode")
+	RootCmd.PersistentFlags().Bool("verbose-credentials", false, "When set, it will not redact the Authorization headers on the request/response trail")
```

Comment:
Given the nature of this flag I think it might be worth it to be a bit more explicit with this text. How about something like:

```suggestion
	RootCmd.PersistentFlags().Bool("verbose-credentials", false, "When set, Authorization headers on the request/response trail will be displayed as plain text")
```